task-2/src/store/apiUrl/index.js

```diff
@@ -0,0 +1,3 @@
+export const moviesApiLink = () => {
+	return `http://localhost:4000/movies?limit=12`;
```

Comment:
This is not necessarily what I meant. Think of it this way: 
what if you want at some point change the limit?
or access `/users` instead of `/movies`?
The "baseUrl" in you case is `http://localhost:4000`.
Then the "moviesEndpoint" is `/movies`.
And then you add the limit.
And it's easier to keep them in an object like:
`const API_CONFIG = {..., baseUrl: '...', moviesEndpoint: '...', ...}`

Reply:
fixed.